pkg/ccl/backupccl/targets.go

```diff
@@ -472,15 +473,17 @@ func MakeBackupTableEntry(
 		ind := -1
 		for i, b := range backupManifests {
 			if b.StartTime.Less(endTime) && endTime.LessEq(b.EndTime) {
-				if endTime != b.EndTime {
-					return BackupTableEntry{}, errors.Newf("should specify the exact backup time, next backup timestamp is %s", b.EndTime.AsOfSystemTime())
+				if endTime != b.EndTime && b.MVCCFilter != MVCCFilter_All {
+					return BackupTableEntry{}, errors.Newf(
```

Comment:
nit: you should use an error hint here and keep the main message short.

See: https://wiki.crdb.io/wiki/spaces/CRDB/pages/1676804870/Error+concepts+and+handling+in+CockroachDB#Error-messages%2C-hints-and-codes

Reply:
Thanks for catching! (I just found github ate my comments somehow